doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -251,16 +245,12 @@ nRF9160 samples
 
 * :ref:`modem_shell_application` sample:
 
-  * Updated:
-
-    * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
-      Remove the inclusion of the file :file:`nrf_cloud_codec.h`.
+  * Updated the sample to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
+    Removed the inclusion of the file :file:`nrf_cloud_codec.h`.
 
 * :ref:`https_client` sample:
 
-  * Added:
-
-    * Add IPv6 support and wait for PDN to fully activate (including IPv6, if available) before looking up address.
+  * Added IPv6 support and wait for PDN to fully activate (including IPv6, if available) before looking up the address.
```

Comment:
This sounds a bit weird. "Added IPv6 support and wait for..."
However, I don't have a good suggestion to make this better.

Reply:
I updated the sentence. Please check.